ee/tabby-webserver/Cargo.toml

```diff
@@ -30,6 +30,8 @@ reqwest = { workspace = true, features = ["json"] }
 rust-embed = "8.0.0"
 serde.workspace = true
 serde_json.workspace = true
+serial_test = "3.0.0"
+serial_test_derive = "3.0.0"
```

Comment:
1. they should be dev dependencies, instead of dependencies.
2. good chance to extract serial test into a workspace dependency to lock version at same time.
<img width="525" alt="image" src="https://github.com/TabbyML/tabby/assets/388154/518cb408-7a00-4241-89da-06ef9c7eeacb">

Reply:
Looks like workspace dependencies can't be dev dependencies